samples/wifi/thread_coex/README.rst

```diff
@@ -186,11 +186,11 @@ Testing
       1050759502         /dev/ttyACM3    VCOM1
       $
 
-	  In this example, ``1050779496`` is the device ID of the first nRF7002 DK and ``1050759502`` is the device ID of the other one.
+In this example, ``1050779496`` is the device ID of the first nRF7002 DK and ``1050759502`` is the device ID of the other one.
 
-	  While connecting to a particular board, use the ttyACMx corresponding to VCOM1.
-	  In the example, use ttyACM1 to connect to the board with device ID ``1050779496``.
-	  Similarly, use ttyACM3 to connect to the board with device ID ``1050759502``.
+While connecting to a particular board, use the ttyACMx corresponding to VCOM1.
+In the example, use ttyACM1 to connect to the board with device ID ``1050779496``.
+Similarly, use ttyACM3 to connect to the board with device ID ``1050759502``.
```

Comment:
There are different ways to write dev/tty in the docs, but none is using plain text, hence the edit.
Similarly, the serial number from the sticker is referred to most often as serial number, while device ID is used more in cloud context (as in glossary, for example).
```suggestion
While connecting to a particular device, use the ``/dev/ttyACM`` corresponding to VCOM1.
In the example, use ``/dev/ttyACM1`` to connect to the device with the serial number ``1050779496``.
Similarly, use ``/dev/ttyACM3`` to connect to the device with the serial number ``1050759502``.
```